goto/commands/cd.py

```diff
@@ -3,7 +3,7 @@
 from ..gotomagic.text import GotoError, GotoWarning
 
 
-def cd(magic, args):
+def cd(magic, command, args):
```

Comment:
👍  for consistency

Reply:
YEs. Can't believe I missed that one

Reply:
You didn't miss it, I just wanted to point out that I love that all the commands have the same parameter signature now :)

Reply:
Oooo.. Yes I see now that it is consistent. Which was the intent, maybe the `cd` command needs to know the command which initiated it, maybe not. The argparser should not care, except for the default edge-case.

Reply:
Actually ... the cd command is never used... 

The cd is done in the `shell-script` layer, before it even hits the python layer. 

That osascript leftovers there, is an early experiment I did on cd-ing, that would only work on osx and it would also open a new terminal window.  

It is safe to delete this command.py